src/site/_includes/partials/head.njk

```diff
@@ -21,6 +21,7 @@
 
 <link rel="preload" as="font" crossorigin href="/fonts/google-sans/regular/latin.woff2">
 <link rel="preload" as="font" crossorigin href="/fonts/google-sans/bold/latin.woff2">
+<link rel="preconnect" href="https://web-dev.imgix.net">
```

Comment:
Minor nit, i like the idea. Can we change the href to `https://{{site.imgixDomain}}`. That way we link it to a var, if it we ever change the domain the preconnect will automatically update.

Reply:
Another minor nit, and I'm not sure it matters: do we need the trailing slash on the URL?

Reply:
`preconnect` will turn the URL passed into an absolute URL and then use the origin to preconnect, so as long as there's enough data to get the origin (in this case there is), it should be fine. https://www.w3.org/TR/resource-hints/#dfn-preconnect